invokeai/frontend/web/src/features/settingsAccordions/components/GenerationSettingsAccordion/GenerationSettingsAccordion.tsx

```diff
@@ -31,7 +31,7 @@ const formLabelProps: FormLabelProps = {
   minW: '4rem',
 };
 
-const badgesSelector = createMemoizedSelector(selectLoraSlice, selectGenerationSlice, (lora, generation) => {
+const badgesSelector = createMemoizedSelector(selectNonDisabledLoraSlice, selectGenerationSlice, (lora, generation) => {
   const loraTabBadges = size(lora.loras) ? [size(lora.loras)] : [];
   const accordionBadges: (string | number)[] = [];
   if (generation.model) {
```

Comment:
I'd prefer to not use an extra selector here - all we need is the count of enabled loras for the badge. So something like this is sufficient:

```ts
const enabledLoRAsCount = filter(lora.loras, (l) => l.isEnabled).length;
const loraTabBadges = enabledLoRAsCount ? [enabledLoRAsCount] : [];
```